### PR TITLE
BIP 2: Promote to Active Status

### PR DESCRIPTION
**Pull request by luke-jr**
This requires rough consensus from the bitcoin-dev mailing list to proceed.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
So this would *forbid* to multi-license where one of the "licenses" is PD?

**Comment by luke-jr** [comment]
PD contradicts licensing, so it doesn't make sense to "dual-license" with it.

**Comment by MarcoFalke** [comment]
Fine with me, just wanted to make sure this is intended.

**Comment by MarcoFalke** [comment]
ACK

**Comment by btcdrak** [comment]
NACK on links to bitcoin.it wiki, BIPs should be self contained (e.g. use this Github repo's wiki).

**Comment by luke-jr** [comment]
@btcdrak The wiki links are for comments only. The BIP itself remains self-contained. (and <insert rant about not having brought this up on the ML, preferably back in early February>)

**Comment by btcdrak** [comment]
@luke-jr Also what are the "comments summary" in the header. That looks like a whole lot of maintenance.

File: bip-0018.mediawiki [hunk 1]
```diff
@@ -2,9 +2,12 @@
   BIP: 18
   Title: hashScriptCheck
   Author: Luke Dashjr <luke+bip17@dashjr.org>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://en.bitcoin.it/wiki/BIP_Comments:BIP_0018
   Status: Draft
   Type: Standards Track
   Created: 2012-01-27
+  License: BSD-3-Clause
```
Comment (btcdrak) [inline bip-0018.mediawiki]: What is the providence of this license?

**Comment by luke-jr** [comment]
Please [read the BIP](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki) and discuss on ML...

**Comment by MarcoFalke** [comment]
> NACK on links to bitcoin.it wiki,

What about only adding them when the accompanying wiki page exists? As I read BIP2, adding those is not mandatory:

> Each BIP should, in its preamble, link to a Bitcoin Wiki page

**Comment by laanwj** [comment]
As this is a process BIP, we're not going to change the status to `Active' like BIP1? 'Final' means it's never meant to change again.

> Some Informational and Process BIPs may also have a status of "Active" if they are never meant to be completed. E.g. BIP 1 (this BIP).

**Comment by luke-jr** [comment]
@laanwj You're right, my bad. This should be ->Active.

**Comment by luke-jr** [comment]
Superceded by #360 since there is irreconcilable disagreement.

**Pull request closed without merging**